locales/en-backend.json5

```
@@ -298,6 +339,27 @@
       female: 'Female',
       male: 'Male',
       nonbinary: 'Non-Binary',
+
+      Group1: '0 - 4 Years',
+      Group2: '5 - 14 Years',
+      Group3: '15 - 34 Years',
+      Group4: '35 - 59 Years',
+      Group5: '60 - 79 Years',
+      Group6: 'Above 80 Years',
+      Total: 'Total',
     },
   },
+  interventions: {
+    'School closure': 'School closure',
```

Match the german Translation using plurals.

```suggestion
    'School closure': 'School closures',
```